src/helpers.js

```diff
@@ -56,6 +88,10 @@ async function createJwt (ims, clientId, imsOrg, techacctEmail, metaScopes, priv
     payload[ims.getApiUrl('/s/' + metaScope)] = true
   }
 
+  const privateKeyFile = getFile(privateKey)
```

Comment:
IF  fs.existsSync THEN fs. readFileSync ...?

Reply:
Not doing sync + avoiding the extra call to `fs.exists` for perf reasons.